setup.sh

```diff
@@ -1 +1,2 @@
-git submodule update --init --recursive --progress
+# We do not longer use submodule for Eigen library
```

Comment:
Can we just delete this file completely?

Reply:
See the discussion above. It will be used again for something else very soon.